### DISCUSSION TITLE
What are the differences between the various formats?

### DISCUSSION THREAD

**Discussion by SleepingPanda** [comictagger/comictagger#564] [Q&A] [answered]
Format: `King Size` and `King-size`, `TPB` and `Trade Paper Back`.

Also, can you explain how should the `# Issues` tag can be used? Is that meant to be used for Volumes/TPBs with multiple issues inside them? If so then can you explain how to use it?

**Comment by lordwelch** [comment]
The format tag is really just a free form text field, the defaults were taken from ComicRack and they just [arbitrarily put some in].
There is no `# Issues` tag. If you are talking about the `issue` tag it then it is the issue number. It's whatever number identifies that issue in the series.

[arbitrarily put some in]: https://web.archive.org/web/20161229183519/http://comicrack.cyolito.com:80/forum/7-general/709-using-comicinfo-xml-in-other-viewers#852

**Reply by SleepingPanda** [reply]
Ah thank you for explaining those.
With regards to `# Issues`,
![HG5IH4EFh0](https://github.com/comictagger/comictagger/assets/5846561/c23ee056-c923-4e64-bf59-ce8054268917)
I'm referring to the tag that's related to this field in the GUI. Sorry for the misunderstanding.

So after filling in some dummy info and saving it, it turns out the tag that's associated with `# Issues` in the GUI is `Count`. This was confusing! I went searching through the wiki and found that according to the [anansi project](https://anansi-project.github.io/docs/comicinfo/documentation#count), `Count` is defined as `The total number of books in the series.`. Does that definition also apply here as well?

**Reply by lordwelch** [reply, marked as answer]
Oh, my bad that is the count or "number" of issues
